components.tsx

```diff
@@ -120,33 +152,56 @@ const Button = (props: {
 }) => (
   <View>
     <TouchableOpacity
-      style={
-        props.variant === "secondary"
-          ? styles.secondaryButtonStyle
-          : styles.buttonStyle
-      }
+      style={styles[`${props.variant || "primary"}Button`]}
       onPress={props.onPress}
       activeOpacity={0.4}
     >
       <AppText
         fontWeight="Medium"
-        style={[
-          props.variant === "secondary" && { textDecorationLine: "underline" },
-          styles.buttonText
-        ]}
+        style={styles[`${props.variant || "primary"}ButtonText`]}
       >
         {props.text}
       </AppText>
     </TouchableOpacity>
   </View>
 );
 
-const TopNavigation = (props: {
+interface BackgroundImageProps {
+  opacity?: number;
+  percentageOfScreenWidth?: number;
+  placement?: "center" | "oneThird";
+}
+const BackgroundImage = (props: BackgroundImageProps) => {
+  const RATIO = 1.5;
```

Comment:
This looks hard-coded for this specific image, but do you really need it?
See if you can get away with out specifying a height by using something like https://reactnative.dev/docs/image#resizemode

Reply:
It's not crucial if you're not able to, but it's a good exercise so that you have a component that could be used with any image

Reply:
eyy, that worked!